client/starwhale/cli/board/board.py

```diff
@@ -122,7 +121,8 @@ async def key_h(self):
 class Models(TableWidget):
     """Models represents starwhale model view"""
 
-    def __init__(self, **kwargs) -> None:
+    # TODO use constance
```

Comment:
`constance` typo?  instance or other words？

Reply:
constant :smiling_face_with_tear: 

changed to `f"{DEFAULT_INSTANCE}/{DEFAULT_PROJECT}"`